aws/vpc_flow_log_monitoring/vpc-flow-log-sam-template.yaml

```diff
@@ -18,7 +18,7 @@ Resources:
       Policies:
         KMSDecryptPolicy:
           KeyId: !Ref KMSKeyId
-      Runtime: python2.7
+      Runtime: python3.7
```

Comment:
Why not 3.8?

Reply:
I saw other lambdas on 3.7 and figured this was safe.